### PR TITLE
Add links to BCFtools howtos

### PR DESCRIPTION
**Pull request by pd3**

### CODE REVIEW

**Comment by jkbonfield** [comment]
It's always good to improve the documentation, so thank you.  Do you have a way to view it via your github account to test what it looks like before merge?

How hard though would it be to merge the bcftools github.io files with this?  Are they totally different renderings (I know both are markdown, but github markdown may differ)?

**Comment by pd3** [comment]
Yes, it can be viewed here https://pd3.github.io/www.htslib.org

I did try to merge it but found working with jekyll markdown a bit limited compared to asciidoctor. So for now only added links to the existing pages and leaving this for later when I, or someone else, feels bored.

**Comment by jkbonfield** [comment]
Thanks for the link.

No problem on the merge thing.  Regular incremental improvements are better than delaying indefinitely for the ideal final goal which we may never get time to do.  I'll add my comments, but will wait until Rob is back before merging anything.

File: _includes/header.html [hunk 1]
```diff
@@ -54,6 +55,9 @@
 						</a>
 						<ul class="dropdown-menu" role="menu">
 							<li><a href="{{ "/support#mailing-lists" | relative_url }}">Mailing Lists</a></li>
+				            <li><a href="https://github.com/samtools/htslib/issues">HTSlib issues</a></li>
```
Comment (jkbonfield) [inline _includes/header.html:58]: Great idea.  Thumbs up from me.

File: doc.md [hunk 2]
```diff
@@ -65,6 +65,28 @@ to consult the documentation for the release you are using.
       [1.1](tabix-1.1.html),
       [1.0](tabix-1.0.html))
 
+## HowTos
+
+[BCFtools](http://samtools.github.io/bcftools/howtos)
+
+- Installation
+    - [Latest development version](http://samtools.github.io/bcftools/)
```
Comment (jkbonfield) [inline doc.md]: Do you really mean http://samtools.github.io/bcftools/ and not http://samtools.github.io/bcftools/howtos/install.html?  Both exist, but differ.

I'm also wondering if we should link to https://github.com/samtools/bcftools/blob/develop/INSTALL or at least update these with that information.  Eg discussion on bzip2 and lzma dependencies (and arguably we could add how to disable these if you don't care about cram support in your bcftools binary).

**Review by jkbonfield — COMMENTED** [review]
I haven't reviewed the text it links to (other than the initial installation bit), but the links themselves look fine barring the one question on installation.

File: doc.md (hunk 2 above)
Comment (pd3) [reply doc.md]: You are right updated now.

**Pull request merged**